Lab1/scripts/deployment.sh

```diff
@@ -72,7 +72,9 @@ if [[ $client -eq 1 ]]; then
   APP_SITE_BUCKET=$(aws cloudformation describe-stacks --stack-name "$stackname" --query "Stacks[0].Outputs[?OutputKey=='AppBucket'].OutputValue" --output text)
   APP_SITE_URL=$(aws cloudformation describe-stacks --stack-name "$stackname" --query "Stacks[0].Outputs[?OutputKey=='ApplicationSite'].OutputValue" --output text)
   APP_APIGATEWAYURL=$(aws cloudformation describe-stacks --stack-name "$stackname" --query "Stacks[0].Outputs[?OutputKey=='APIGatewayURL'].OutputValue" --output text)
-
+  
+  # remove trailing slash from url
+  APP_APIGATEWAYURL=${APP_APIGATEWAYURL%/}
```

Comment:
Actually in the Cloudformation template, output value of these api gateway url output parameters have a trailing slash appended. I think we should remove the trailing slash for those output parameters in the cloudformation templates. You need to update server/shared-template.yaml and server/tenant-template.yaml files in each lab.